api/api.js

```diff
@@ -46,6 +46,19 @@ export const fetchUserStats = async (bodyObject) => {
   return fetch(`${ENDPOINT}/account/donations`, constructPostBody(bodyObject))
 }
 
+export const fetchBlog = _ => {
```

Comment:
We don't know if we will be serving the blog content through the backend API or via static files in this repo. Adding API client logic at this point is premature. Please remove this and others fake integrations (I will comment separately on the other parts for clarity).

Reply:
Thanks for reviewing this code... As per my comment on `https://github.com/teacherfund/TeacherFund_next/issues/125#issuecomment-540149798` I assumed things to be fetched from API... 
Also, server a larger number of blog post from static files is not a good practice! At least I assume so... 
Though I would change the fake supply data location for now...

Reply:
Please refer to `pages/blog.js` file where I'm importing from `../components/Blog` not `../components/Blog/index`... Additionally, `index` file acts as an encapsulation for that particular module as other modules don't need to know about the exact files in that particular module where I'm exporting the complete component... In this case `component/blog/blogPage` acts as entry files... If other modules wants to use my blog component they need to refer to this particular module `component/blog/blogPage` which is a hassle for large number of components

Reply:
In the latest commits, I changed it to use static content.. Pls, take a look